<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportTariffs</class>
 <widget class="QDialog" name="ImportTariffs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Загрузить из</string>
     </property>
     <property name="buddy">
      <cstring>edtFileName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="edtFileName"/>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btnSelectFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QTextBrowser" name="log"/>
   </item>
   <item row="12" column="0" colspan="4">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Начать импорт</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QCheckBox" name="chkFullLog">
     <property name="text">
      <string>Подробный отчет</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QSpinBox" name="edtOrgInfis">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblTariffType">
     <property name="text">
      <string>Тип тарифа</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblOrgInfis">
     <property name="text">
      <string>Код ЛПУ</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblOrgLevel">
     <property name="text">
      <string>Уровень ЛПУ</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QComboBox" name="cmbOrgLevel">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QComboBox" name="cmbTariffType"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>Тип события</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtFileName</tabstop>
  <tabstop>btnSelectFile</tabstop>
  <tabstop>chkFullLog</tabstop>
  <tabstop>log</tabstop>
  <tabstop>btnImport</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
